src/LeafletService.php

```diff
@@ -50,6 +50,15 @@ public function getParameterInfo(): array {
 			'islist' => true,
 		];
 
+		$params['layers_dark'] = [
```

Comment:
To match the naming scheme of the other parameters this should be `darklayers`, perhaps with an added `dark_layers` alias.